CHANGELOG.md

```diff
@@ -1,4 +1,319 @@
+(kernel or iptables versions are insufficient), the kube-proxy will fall-back
+to userspace mode. Kube-proxy is much more performant and less
+resource-intensive in ‘iptables’ mode. 
+  * Node stability can be improved by reserving [resources](https://github.com/kubernetes/kubernetes/blob/release-1.2/docs/proposals/node-allocatable.md) for the base operating system using --system-reserved and --kube-reserved Kubelet flags
+  * Liveness and readiness probes now support more configuration parameters:
+periodSeconds, successThreshold, failureThreshold
+  * The new ReplicaSet API (Beta) in the Extensions API group is similar to
+ReplicationController, but its [selector](http://kubernetes.io/docs/user-guide/labels/#label-selectors) is more general (supports set-based selector; whereas ReplicationController
+only supports equality-based selector).
+  * Scale subresource support is now expanded to ReplicaSets along with
+ReplicationControllers and Deployments. Scale now supports two different types
+of selectors to accommodate both [equality-based selectors](http://kubernetes.io/docs/user-guide/labels/#equality-based-requirement) supported by ReplicationControllers and [set-based selectors](http://kubernetes.io/docs/user-guide/labels/#set-based-requirement) supported by Deployments and ReplicaSets.
+  * “kubectl run” now produces Deployments (instead of ReplicationControllers) and
+Jobs (instead of Pods) by default.
+  * Pods can now consume Secret data in environment variables and inject those
+environment variables into a container’s command-line args.
+  * Stable version of Heapster which scales up to 1000 nodes: more metrics, reduced
+latency, reduced cpu/memory consumption (~4mb per monitored node).
+  * Pods now have a security context which allows users to specify:
+     * attributes which apply to the whole pod:
+        * User ID
+        * Whether all containers should be non-root
+        * Supplemental Groups
+        * FSGroup - a special supplemental group
+        * SELinux options
+     * If a pod defines an FSGroup, that Pod’s system (emptyDir, secret, configMap,
+etc) volumes and block-device volumes will be owned by the FSGroup, and each
+container in the pod will run with the FSGroup as a supplemental group
+  * Volumes that support SELinux labelling are now automatically relabeled with the
+Pod’s SELinux context, if specified
+  * A stable client library release\_1\_2 is added. The library is [here](https://github.com/kubernetes/kubernetes/tree/master/pkg/client/clientset_generated/release_1_2), and detailed doc is [here](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/generating-clientset.md#released-clientsets). We will keep the interface of this go client stable.
+  * New Azure File Service Volume Plugin enables mounting Microsoft Azure File
+Volumes (SMB 2.1 and 3.0) into a Pod. See [example](https://github.com/kubernetes/kubernetes/blob/release-1.2/examples/azure_file/README.md) for details.
+
```

Comment:
We are missing one notable improvement on monitoring side: Better disk usage monitoring by adding support for logs usage of containers, root filesystem usage of containers, host based pod volumes and node

How about including something like:
```
Logs usage and root filesystem usage of a container, volumes usage of a pod and node disk usage are exposed through Kubelet new metrics API
```

I know heapster mentioned more metrics. But again we have the users directly scraping kubelet API for stats / metrics.